Mage.Sets/src/mage/cards/s/ScarwoodBandits.java

```diff
@@ -0,0 +1,134 @@
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.keyword.ForestwalkAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetArtifactPermanent;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ *
+ * @author L_J
+ */
+public class ScarwoodBandits extends CardImpl {
+
+    public ScarwoodBandits(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.CREATURE},"{2}{G}{G}");
+        this.subtype.add("Human");
+        this.subtype.add("Rogue");
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(2);
+
+        // Forestwalk
+        this.addAbility(new ForestwalkAbility());
+        
+        // {2}{G}, {tap}: Unless an opponent pays {2}, gain control of target artifact for as long as Scarwood Bandits remains on the battlefield.
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new ScarwoodBanditsEffect(new GenericManaCost(2)), new ManaCostsImpl("{2}{G}"));
+        ability.addCost(new TapSourceCost());
+        ability.addTarget(new TargetArtifactPermanent());
+        this.addAbility(ability);
+    }
+
+    public ScarwoodBandits(final ScarwoodBandits card) {
+        super(card);
+    }
+
+    @Override
+    public ScarwoodBandits copy() {
+        return new ScarwoodBandits(this);
+    }
+}
+
+class ScarwoodBanditsEffect extends OneShotEffect {
+
+    protected Cost cost;
+
+    public ScarwoodBanditsEffect(Cost cost) {
+        super(Outcome.Detriment);
+        this.staticText = "Unless an opponent pays {2}, gain control of target artifact for as long as Scarwood Bandits remains on the battlefield";
+        this.cost = cost;
+    }
+
+    public ScarwoodBanditsEffect(final ScarwoodBanditsEffect effect) {
+        super(effect);
+        this.cost = effect.cost.copy();
+    }
+
+    @Override
+    public ScarwoodBanditsEffect copy() {
+        return new ScarwoodBanditsEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        if (controller != null) {
+            Permanent targetArtifact = game.getPermanent(getTargetPointer().getFirst(game, source));
+            if (targetArtifact != null) {
+                Player player = game.getPlayer(targetArtifact.getControllerId());
+                if (player != null) {
+                    cost.clearPaid();
+                    if (player.chooseUse(Outcome.Benefit, "Pay {2}? (Otherwise Scarwood Bandits' controller gains control of" + targetArtifact.getLogName() + ")", source, game)) {
+                        cost.pay(source, game, targetArtifact.getControllerId(), targetArtifact.getControllerId(), false, null);
```

Comment:
This should actually ask each opponent if they want to pay {2} and allow them to pay even if someone else already has.